docs/administration-guide/gaffer-deployment/gaffer-docker/gaffer-images.md

```diff
@@ -0,0 +1,111 @@
+
+!!! note
+    There are a few other images available; however, they are less frequently
+    used or purely example images, please see the [`gaffer-docker`](https://github.com/gchq/gaffer-docker/tree/develop/docker)
+    repository for more details.
+
+## Volumes and bind-mount
+
+To change and configure the graph that is deployed you will need to override
+the default files in the images by default. You can of course create a custom
+image with different config files however, it can be more flexible to just
+bind-mount over the current files.
+
+To do this you will need to know the location of the files in the image you
+want to override but in many cases you can mount over an entire directory
+for example:
+
+!!! example ""
+    The path `/custom/configs` is some path on the host system with different
+    config files in that can be mounted in when running the image.
+
+    ```bash
+    docker run \
+           -p 8080:8080 \
+           -v /custom/configs/gaffer/graph:/gaffer/graph \
+           -v /custom/configs/gaffer/schema:/gaffer/schema \
+           -v /custom/configs/gaffer/store:/gaffer/store \
+           gchq/gaffer-rest:2.0.0
+    ```
+
+## Custom Images
+
+To avoid managing a file on the host and bind-mount it, the configuration can be
+baked into the image. This works well if the configuration itself is rather
+static and the same across all environments.
+
+Creating a custom image can also be useful if you want to load custom extensions
+to use with Gaffer (e.g. Jars) by default.
+
+To create a custom image simply make a new `Dockerfile` and use one of the Gaffer
+images as the base image like the following:
+
+```dockerfile
+FROM gchq/gaffer-rest:latest
+
+# Copy over the existing directory with store configs in
+COPY ./custom/configs/gaffer/store /gaffer/store
+```
+
+Then build the new image using a suitable tool or just plain Docker from the
+current directory like:
+
+```bash
+docker build -t my-gaffer-rest .
+```
+
+### Adding Additional Libraries
+
+By default with the Gaffer deployment you get access to the:
+
+- Sketches library
+- Time library
+- Bitmap Library
+- JCS cache library
+
+If you want more libraries than this (either one of ours of one of your own) you
+will need to customise the docker images and use them in place of the defaults.
+
+At the moment, the `gchq/gaffer-rest` image uses a runnable jar file located at
+`/gaffer/jars`. When it runs it includes the `/gaffer/jars/lib` on the
+classpath. There is nothing in there by default because all the dependencies are
+bundled in to the JAR. However, if you wanted to add your own jars, you can add
+then to this directory like the following:
+
+```dockerfile
+FROM gchq/gaffer-rest:latest
+COPY ./custom-lib:1.0-SNAPSHOT.jar /gaffer/jars/lib/
+```
+
+To add any libraries to the `gchq/gaffer` image in order to push down any extra
+value objects and filters to Accumulo you have to add the jars to the
```

Comment:
I'm not sure what's meant by "push down any extra value objects and filters". This directory is used for JARs to add to the Accumulo classpath. Either new iterators or dependencies for them. There's more detail about this on the Accumulo migration page as this path differs between Accumulo 1 and 2. If this was copied from elsewhere it needs to be updated.

Reply:
This paragraph I believe was in the existing k8s docs that I moved here, I agree the wording is a little strange I'll simplify and add a link to the migration page